resources/views/store/product.blade.php

```diff
@@ -131,13 +131,39 @@ class="js-gallery"
             @endif
         </div>
 
-        @if($product->inStock())
         <div class="osu-layout__sub-row osu-layout__sub-row--with-separator" id="add-to-cart">
             <div class="big-button">
-                <button type="submit" class="js-store-add-to-cart btn-osu btn-osu-default">Add to Cart</button>
+                <button type="submit" class="js-store-add-to-cart btn-osu btn-osu-default"
+                    @if(!$product->inStock())
```

Comment:
sorry, completely forgot that i simply reload the page after processing the request :confused: